e2e/provision/playbooks/mirror_git_repo_to_gitea.yaml

```diff
@@ -0,0 +1,57 @@
+- name: "Create gitea repo: {{ item.name }}"
+  kubernetes.core.k8s:
+    context: kind-kind
+    state: present
+    definition:
+      apiVersion: infra.nephio.org/v1alpha1
+      kind: Repository
+      metadata:
+        name: "{{ item.name }}"
+        namespace: default
+      spec:
+        description: "{{ item.name }} repository"
+        defaultBranch: main
+
+- name: "Pull GitHub repo {{ item.name }}"
+  ansible.builtin.git:
+    repo: "{{ item.repo_uri }}"
+    dest: "/tmp/repository/{{ item.name }}.git"
+    bare: yes
+
+- name: "Add gitea remote to local git repo"
+  ansible.builtin.shell: |
+    cd /tmp/repository/{{ item.name }}.git ; git remote add gitea http://nephio:secret@172.18.0.200:3000/nephio/{{ item.name }}.git
+
+- name: "Push tags to gitea: {{ item.name }}"
+  ansible.builtin.shell: |
+    cd /tmp/repository/{{ item.name }}.git ; no_proxy="$no_proxy,172.18.0.200" NO_PROXY="$NO_PROXY,172.18.0.200"  git push gitea --tags --force
```

Comment:
```suggestion
  environment:
    no_proxy: "{{ gitea_ip}}"
    NO_PROXY: "{{ gitea_ip}}"
  ansible.builtin.command:
    cmd: git push gitea --tags --force
    chdir: /tmp/repository/{{ item.name }}.git
```

Reply:
again, much nicer this way. I will change it